CMakeLists.txt

```diff
@@ -29,6 +36,11 @@ if(NOT CMAKE_BUILD_TYPE MATCHES "^(Debug|Release|RelWithDebInfo|MinSizeRel)$")
                                                "MinSizeRel" "RelWithDebInfo")
 endif()
 
+# Set compiler flags.
+if(CMAKE_Fortran_COMPILER_ID MATCHES "^(Intel)$")
+  set(CMAKE_Fortran_FLAGS_DEBUG "-O0 -check all -ftrapuv")
+endif()
+
```

Comment:
Its better to move it down in the `ncep_post.fd/CMakeLists.txt`.
See suggestion below.
```suggestion
```